docs/conf.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# Configuration file for the Sphinx documentation builder.
+# See https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+copyright = "2020-2024, Quantinuum"
+author = "Quantinuum"
+
+
+html_title = "pytket user guide"
+
+extensions = [
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.mathjax",
+    "jupyter_sphinx",
+    "sphinx_copybutton",
+    "sphinx.ext.autosectionlabel",
+    "myst_nb",
+]
+
+myst_enable_extensions = ["dollarmath", "html_image", "attrs_inline"]
+
+html_theme = "furo"
+templates_path = ["./quantinuum-sphinx/_templates/"]
+html_static_path = ["./quantinuum-sphinx/_static/", "_static/"]
+
+pytketdoc_base = "https://tket.quantinuum.com/api-docs/"
+
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+    "pytket": (pytketdoc_base, None),
+    "qiskit": ("https://docs.quantum.ibm.com/api/qiskit", None),
+    "pytket-qiskit": ("https://tket.quantinuum.com/extensions/pytket-qiskit/", None),
```

Comment:
Why do you have only two extensions in here?

Reply:
These are just the two most common  ones. I can add in some more.